tests/publiccloud/azure_aitl.pm

```diff
@@ -75,16 +80,22 @@ sub run {
     my $status_data;
     while (1) {
         # Get the current job status
-        my $status = script_output(qq(python3.11 /tmp/aitl.py job get $aitl_get_options -q "properties.results[].status|{RUNNING:length([?@=='RUNNING']),QUEUED:length([?@=='QUEUED']),ASSIGNED:length([?@=='ASSIGNED']),FAILED:length([?@=='FAILED'])}"));
+        my $status = script_output(qq($aitl_job get $aitl_get_options -q "properties.results[].status|$monitoring"));
+
+        if ($status =~ /no result returned/ig) {
+            sleep(60);
+            record_info("WARN:", "no results:\n" . $status);
+            next;
+        }
 
-        # Remove the first two non-JSON lines from the status JSON
+        # Remove the first two/3 non-JSON lines from the status JSON
```

Comment:
mmm what `\3` suppose to mean ? or it is just a typo ?

Reply:
It is referring to the espression {1,3}, already present to clean the result able to remove up to 3 lines; but also referring to this issue case, where the initial logs are 3 text lines(only), and the normal case are 2 text lines plus 1 blank (total 3), removed by this command.